libraries/AP_Motors/AP_Motors_Class.h

```diff
@@ -181,9 +181,10 @@ class AP_Motors {
     // set loop rate. Used to support loop rate as a parameter
     void                set_loop_rate(uint16_t loop_rate) { _loop_rate = loop_rate; }
 
-    // return the roll factor of any motor, this is used for tilt rotors and tail sitters
+    // return the roll or pitch factor of any motor, this is used for tilt rotors and tail sitters
     // using copter motors for forward flight
     virtual float       get_roll_factor(uint8_t i) { return 0.0f; }
+    virtual float       get_pitch_factor(uint8_t i) { return 0.0f; }
```

Comment:
Using pitch factor makes no sense on tilt-rotors, its no longer valid once the motors tilt. To be honest I would rather add a option bitmask to allow Qassist in acro mode than do it this way. This only gets you manual on top of that.

Reply:
what about the case when you dont use all the motors in forward flight....is this smart enough to know if you have a setup that will allow differential pitch? like only using the top two motors as forward motors on a quad?

Reply:
@IamPete1 definitely needs more work for tiltrotors; should either disable differential pitch for tilting motors or take tilt into account for both roll and pitch factors.
@Hwurzburg If no motors with a non-zero pitch factor are enabled in FF, then this will have no effect. There could be a pre-arm message to warn about that, but it's more a documentation issue, I think. 
 
Has anyone built a copter tailsitter with tilting motors???  I think we only support tilt-vectoring in dual-motor tailsitter configuration, but there's nothing stopping people from tilting the motors on a quad tailsitter... instead of a fixed 90 degrees, the tailsitter rotation angle could be a continuous variable :)